.gitignore

```diff
@@ -1,2 +1,3 @@
 .cm
 .smackage
+bin
```

Comment:
there's actually an important shell script in `bin` right now that is called by the `makefile`. Perhaps we should move it...